go.mod

```diff
@@ -1,19 +1,12 @@
-module github.com/micromdm/scep/v2
+module github.com/smallstep/scep/v2
```

Comment:
:thinking: should _we_ start with `v2`?

Reply:
I think it makes sense, yes. The current functionality is backwards and drop-in compatible. I think this makes it simpler to explain to users that currently import `micromdm/scep/v2` that they can simply replace it with our `v2`. Then our next version, which I see happening very soon, will be `v3`, which will then be a logical and spiritual successor to both our `v2` as well as MicroMDM's. I've seen this pattern before, for example with `go-jose`: moved to a different organization, but continued numbering.

Reply:
As far as I see, we are only moving `github.com/micromdm/scep/v2/scep`, I don't think there is a need for `v2` at the moment, the replacement will be quite similar, and I think it's more confusing than anything else. Especially because we won't have a `v1`.

We should also change this repo to be a source, not a fork. If not by default, GitHub will should you a massive merge when you do a PR on this repo.